docs/index.md

```diff
@@ -135,14 +135,14 @@ Next, define a set of services using `docker-compose.yml`:
     redis:
       image: redis
 
-This defines two services:
+This template defines two services:
```

Comment:
@charleswhchan There is a problem in this section.  The section is using bullets and headings when sentences alone are better.  

As a maintainer, you should be able to edit my comment and snag this content.

----->
This template defines two services, `web` and `redis`. The `web` service:

* Builds from the `Dockerfile` in the current directory.
* Forwards the exposed port 5000 on the container to port 5000 on the host machine.
* Mounts the current directory on the host to `/code` inside the container allowing you to modify the code without having to rebuild the image.
* Links the web container to the Redis service.

The `redis` service uses the latest public [Redis](https://registry.hub.docker.com/_/redis/) image pulled from the Docker Hub registry.